emva1288/process/routines.py

```diff
@@ -42,17 +42,23 @@ def get_int_imgs(imgs):
 
     sum_ = 0
     sq_ = 0
+    n = 0
+    dmean_ = 0
     for img in imgs:
         # we force the images as int64 to make sure we do not clip
         i = img.astype(np.int64)
         sum_ += i
         sq_ += np.square(i)
+        if n > 0:
+            dmean_ += np.square(np.mean(i) - np.mean(oi))
```

Comment:
Based on your naming scheme, `dmean_` I assume means "delta mean", not the square of the delta mean.  It might be clearer to do the squaring later when you actually need it to correct the variance (line 129 of data.py)